kbase-extension/static/kbase/js/common/errorDisplay.js

```diff
@@ -0,0 +1,197 @@
+define(['bluebird', 'uuid', 'common/ui', 'common/html'], (Promise, Uuid, UI, html) => {
```

Comment:
This was under `nbextensions/appCell2/widgets/tabs/errorTab.js`, but I moved it to `common` since  there is nothing specific to the app cell in it. Added tests for this module.